### ISSUE TITLE
Kernel error AppleACPIPlatform

### ISSUE THREAD

**Issue by AlinEusebiu98** [PIut02/ROG-Zephyrus-G14-GA401-Hackintosh#31]
### Discussed in https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/discussions/30

<div type='discussions-op-text'>

<sup>Originally posted by **AlinEusebiu98** February 10, 2024</sup>
Hello guys, I did try this with my 2021 GA401Q, I do get the same error every time I tryed, I did try a clean EFI made from 0, I did try this one and change the settings that you suggested in the readme part.
I did try with the recovery method (Monterey) and with the Sonoma.raw 14.x but never got to the installation part, I know it's something with the booting part, but can't figure it out, I did try everyting for the past 3 days...

**My specs:**
GA401QE.415 
Ryzen 7 5800HS with Vega 8
16GB RAM DDR4 = Samsung M471A1G44AB0
Nvidia RTX 3050Ti 4GB
SSD: SAMSUNG MZVLQ512HBLU
Wi-Fi: MediaTek MT7921

![WhatsApp Image 2024-02-10 at 11 36 15_8a83d0ea](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/74622198/65a806dc-048b-489c-bb17-e45d494a8780)
![WhatsApp Image 2024-02-10 at 11 41 38_b6a074fb](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/74622198/8c51f251-a2d4-4397-85a7-25539ddf30ca)

**Comment by ngntriminh** [comment]
Please attach your EFI folder here so people can give it a look and help you.

**Comment by AlinEusebiu98** [comment]
Thank you!
I did try the WhatEverGreen.kext too(I disabled the NotedRed), dind't work. So you will see both of them in the Kext folder.
https://fastupload.io/e5uy9d3OIM8r043/file

**Comment by AlinEusebiu98** [comment]
Hi! Thank you so much for your time!
I did everything that you said, now it does get stuck here and after this it's rebooting! 
![image](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/74622198/7f98179c-eaa9-4e0b-9da9-2d68e576f30e)

**Comment by ngntriminh** [comment]
> Hi! Thank you so much for your time! I did everything that you said, now it does get stuck here and after this it's rebooting! ![image](https://private-user-images.githubusercontent.com/74622198/305374807-7f98179c-eaa9-4e0b-9da9-2d68e576f30e.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgwNzc5ODMsIm5iZiI6MTcwODA3NzY4MywicGF0aCI6Ii83NDYyMjE5OC8zMDUzNzQ4MDctN2Y5ODE3OWMtZWFhOS00ZTBiLTlkYTktMmQ2OGU1NzZmMzBlLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxNlQxMDAxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOWEzY2QyYzIxNTdlYjBmZGI4MWMyMTUzZmNjM2MyMGY1ZjAzMTU1M2RkNjVhZjM1YzY3MzFmMDhhNzkxMmZlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.vRxq4FdkmAvNlSwTjKUKWd58lig1VKn5RI35bFELj9Q)

Please try again a few times and wait. Sometimes I also have to wait for a decent time to move to the next step. Maybe `RESET NVRAM` again could help you. I think you need to `RESET NVRAM` in a proper way, if not, please go to your config file > _Misc_ > _Boot_ > disable _HideAuxiliary_ and go to _Misc_ > _Tools_ then disable all the tools available there. After that, when you're at the _Opencore_ Boot Menu, you'll see an option to `RESET NVRAM` (and an other option to disable _SIP_ if you want to). Just select that option and try again.

**Comment by AlinEusebiu98** [comment]
I did, know it s rebooting after acpi.
![image](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/74622198/051b1855-3aad-419c-81a6-a66cae5d9230)

**Comment by ngntriminh** [comment]
> I did, know it s rebooting after acpi. ![image](https://private-user-images.githubusercontent.com/74622198/305383275-051b1855-3aad-419c-81a6-a66cae5d9230.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgwNzk1MzYsIm5iZiI6MTcwODA3OTIzNiwicGF0aCI6Ii83NDYyMjE5OC8zMDUzODMyNzUtMDUxYjE4NTUtM2FhZC00MTljLTgxYTYtYTY2Y2FlNWQ5MjMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE2VDEwMjcxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIwNWM1M2M3NmFmNTYwNzBkN2NlZGNjY2ZjMjY2OGU0MjIwN2E5NWIxYmMzZmY4OGY5YWRlYWM3NmYxYTBjNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.H5kDx5BWr1jBraQUX9uCSWRZwjawDC0VCPYIT3GDIX4)

In my case, I don't usually face bootloops, sometimes I am stuck and I just wait until it continues booting. So if you get stuck, be patient for a while to see if it could continue. In addition, have you disabled _Fast Boot_ and _Secure Boot_ in your _BIOS_? If you've already turned them off, could you try again a few times? If you still face bootloops it means your EFI needs to be fixed.

**Comment by PIut02** [comment]
I still think this issue is caused by the SSD, as can be seen on other forums with a lot of these models of Samsung SSDs causing various issues.  If you want to use a hackintosh then it is not recommended to use a Samsung SSD.

**Comment by ngntriminh** [comment]
> I still think this issue is caused by the SSD, as can be seen on other forums with a lot of these models of Samsung SSDs causing various issues. If you want to use a hackintosh then it is not recommended to use a Samsung SSD.

Yeah, maybe you're right. I've also seen a person who had the Samsung SSD and he faced a similar issue and then posted it here. He successfully hackintoshed his laptop by trying on a different SSD which is not Samsung. It's worth trying that @AlinEusebiu98 if you still want to hackintosh yours after facing failures although you try my previous recommendations.

**Comment by AlinEusebiu98** [comment]
Thank you so much for your time guys. I will try to change the SSD next week and also the network card and I will come back here to let you know what did happen. thank you so much again.

**Comment by AlinEusebiu98** [comment]
What SSD should I buy? 
What about my ram? It s 16gb oke?

**Comment by PIut02** [comment]
> What SSD should I buy? 
> What about my ram? It s 16gb oke? 

I'm using Western Digital sn570 for your reference, and I think 16gb of ram is enough.

**Comment by AlinEusebiu98** [comment]
Oke, thank you!
